site/static/examples/kind-with-registry.sh

```diff
@@ -23,7 +23,10 @@ EOF
 
 # connect the registry to the cluster network
 # (the network may already be connected)
-docker network connect "kind" "${reg_name}" || true
+kindnetwork=$(docker inspect --format='{{json .NetworkSettings.Networks.kind}}' "${reg_name}")
```

Comment:
nit: we're only going to compare this value, so we should just compare the result directly.
```suggestion
if [ "$(docker inspect --format='{{json .NetworkSettings.Networks.kind}}' "${reg_name}")" = 'null' ]; then
```

also, we should single-quote literals (`'null'`)

Reply:
actually, let's just do the single quote nit for now, I don't know why we didn't compare directly above. perhaps trying to avoid scrolling in the docs presentation?

Reply:
we can always revisit both conditionals later

Reply:
Oops, pushed update while you were still looking. Let me know if the latest rev needs any changes.